src/main/java/base/gameobjects/services/AnimalNamingService.java

```diff
@@ -0,0 +1,58 @@
+package base.gameobjects.services;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.*;
+
+import static base.constants.Constants.LANGUAGE;
+import static base.constants.FilePath.FEMALE_NAMES_FILE_PATH;
+import static base.constants.FilePath.MALE_NAMES_FILE_PATH;
+
+public class AnimalNamingService {
+
+    private final List<String> femaleNamesList = new ArrayList<>();
+    private final List<String> maleNamesList = new ArrayList<>();
+    private final Random random = new Random();
+
+    public AnimalNamingService(){
```

Comment:
I noticed you have inconsistent spacings in the code. If you are using Intellij IDEA, just press ctrl+alt+L and whole file will autoformat.